docs/swagger/spec.yml

```diff
@@ -2331,7 +2447,73 @@ definitions:
       maxModifiedDatetime:
         type: string
         format: datetime
+  UploadCreate:
+    type: object
+    properties:
+      uploadStatus:
+        type: string
+        description: Status of upload
+        enum:
+          - CREATED
+          - UPLOADING
+          - UPLOADED
+          - QUEUED
+          - PROCESSING
+          - COMPLETE
+      files:
+        type: array
+        items:
+          type: string
+      uploadType:
+        type: string
+        description: Source of files and uploads
+        enum:
+          - DROPBOX
+          - S3
+          - LOCAL
+      fileType:
+        type: string
+        description: type of data being uploaded, limited to two options right now
+        enum:
+          - GEOTIFF
+          - GEOTIFF_WITH_METADATA
+      datasource:
+        type: string
+        description: datasource of tiffs being uploaded
+        format: uuid
 
+  Upload:
+    allOf:
+    - $ref: '#/definitions/BaseModel'
+    - $ref: '#/definitions/UserTrackingMixin'
+    - $ref: '#/definitions/UploadCreate'
+    - type: object
+      properties:
```

Comment:
Looking at the official spec http://swagger.io/specification/#models-with-composition-90 they seem to be including `properties` under `allOf`. I think this could need more research. I think we should merge what we have for this card, and make a new task for looking into it.